pilot/cmd/pilot-agent/app/cmd.go

```diff
@@ -26,6 +26,7 @@ import (
 	"istio.io/istio/pilot/cmd/pilot-agent/config"
 	"istio.io/istio/pilot/cmd/pilot-agent/options"
 	"istio.io/istio/pilot/cmd/pilot-agent/status"
+
```

Comment:
minor nit: there should not be a space between the import statements since they are all part of the same `istio.io` org